### ISSUE TITLE
verifiy device number

### ISSUE THREAD

**Issue by jmt85** [Qabel/qabel-android#472]
There must be a function in which you can enter the validation code, from the sms, in the app. 
After sending the correct code the corresponding device number should be unlocked

**Comment by jan-schreib** [comment]
I don't know how much you guys talked about this, but the anonymity is basically gone when you do that.

**Comment by audax** [comment]
This feature is for validating your phone number for the register server. The register servers only purpose is to remove the anonymity so that people can find each other in Qabel without exchanging their contact data in person.

The whole register server stuff is optional and meant for the "public identity" to establish first contact.

**Comment by mr-gosh** [comment]
We should make an investigation how many providers are present for such a function.

I know twilio
https://www.twilio.com/sms/pricing/de

But perhaps there are others or espacially privacy aware providers?

**Comment by thechauffeur** [comment]
@jan-schreib: Exactly. This service purpose is to provide UX feeling like you got with e.g. signal but is totally optional.

**Comment by mr-gosh** [comment]
after the identity creation wizard the user should be seamlessly guided through the index registration like in these mockups:

just as a reminder, this is the last step if the id-creation wizard:
![identity-creator-5](https://cloud.githubusercontent.com/assets/3028703/17394240/533faf30-5a28-11e6-8a59-a15cdc9a24ef.png)

in the first step you can register a mail address:
![identity-creator-6](https://cloud.githubusercontent.com/assets/3028703/17394260/7187a010-5a28-11e6-8fc1-afc6f0416ff1.png)

if you want to know more about the drawbacks mentioned by @jan-schreib there should be an info box with further information:
![identity-creator-6-info](https://cloud.githubusercontent.com/assets/3028703/17394283/9b95d9bc-5a28-11e6-852a-6c58bc08e4fa.png)

the next step would be the registration of an mobile phone number:
![identity-creator-7](https://cloud.githubusercontent.com/assets/3028703/17394306/bb22512a-5a28-11e6-9dee-085b812264f1.png)

than there sould be a final updated summary page:
![identity-creator-8](https://cloud.githubusercontent.com/assets/3028703/17394331/d76fa1fc-5a28-11e6-8bd2-f356c2bab4fd.png)

to edit these things there need to be an option in the "edit identity" view:
![identity-editor](https://cloud.githubusercontent.com/assets/3028703/17394360/fc0291a0-5a28-11e6-942a-c1d9722a8b0e.png)

did i miss something?
